lightkurve/targetpixelfile.py

```diff
@@ -762,22 +765,37 @@ def plot(self, ax=None, frame=0, cadenceno=None, bkg=False, aperture_mask=None,
             except IndexError:
                 raise ValueError("cadenceno {} is out of bounds, "
                                  "must be in the range {}-{}.".format(
-                                     cadenceno, self.cadenceno[0], self.cadenceno[-1]))
+                                     cadenceno, self.cadenceno[0], self.cadenceno[-1]))      
```

Comment:
Remove trailing whitespace:
```suggestion
                                     cadenceno, self.cadenceno[0], self.cadenceno[-1]))
```